src/keri/app/apping.py

```diff
@@ -28,9 +28,9 @@ def __init__(self, db=None, console=None, **kwa):
         self.db = db if db is not None else basing.Baser()
         self.console = console if console is not None else serialing.Console()
 
-    def enter(self):
+    def enter(self, *, temp=None):
         """"""
-        if not self.console.reopen():
+        if not self.console.reopen(): # reopen(temp=temp)
             raise IOError("Unable to open serial console.")
```

Comment:
Why is this comment here?

Reply:
I was mirroring Sam's comments in his PR. The comment is now removed.